### PR TITLE
Add gcc compiler activation scripts

### PR DESCRIPTION
**Pull request by isuruf**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge/staged-recipes, @conda-forge/arm-arch, @conda-forge/help-ppc64le, ready for review

File: recipes/ctng-compiler-activation/activate-binutils.sh [hunk 1]
```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
```
Comment (scopatz) [inline recipes/ctng-compiler-activation/activate-binutils.sh]: Should be `/usr/bin/env bash`

Comment (isuruf) [reply recipes/ctng-compiler-activation/activate-binutils.sh]: Activate scripts are sourced and it doesn't matter. This script supports both `bash` and `tsh`

**Comment by jjhelmus** [comment]
LGTM. Great to see this getting out of the `c4aarch64` channel and into `conda-forge`

**Comment by isuruf** [comment]
@jjhelmus, merge?

**Comment by isuruf** [comment]
Thanks

**Pull request merged**